packages/cli/tsup.config.ts

```diff
@@ -5,5 +5,8 @@ export default defineConfig((options: Options) => ({
   format: ['esm'],
   clean: !options.watch,
   sourcemap: true,
+  env: {
+    CLI_SEGMENT_WRITE_KEY: process.env.CLI_SEGMENT_WRITE_KEY ?? 'not-a-valid-segment-write-key',
```

Comment:
Why do we set this erroneous default?

Reply:
You're right that we can probably avoid it here, but it's ultimately because `writeKey` must not be `undefined` when instantiating `Analytics`

Reply:
It's because `env` is typed as:
```
env?: {
    [k: string]: string;
} | undefined
```